generators/import-jdl/index.js

```diff
@@ -154,42 +143,57 @@ module.exports = class extends BaseGenerator {
                     }
                     this.error(`Error while parsing applications and entities from the JDL ${error}`);
                 }
+            }
+        };
+    }
+
+    get install() {
```

Comment:
@MathieuAA not sure about doing the application generation in the install phase. I think it might be better to do the generation in the default phase. Any specific reason for this?
The install phase will queue up along with the install phase of all the sub gen that are composed from this as well and might cause race conditions

Reply:
According to the doc, being in the install scope is recommended when using the spawnCommand